src/script/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -173,7 +173,7 @@ export const RichTextEditor = ({
             openStateRef={mentionsOpen}
           />
 
-          <OnChangePlugin onChange={saveDraft} />
+          <OnChangePlugin onChange={saveDraft} ignoreSelectionChange />
```

Comment:
This is a nice little optimization to avoid updating the stored value everytime the selection changes